### PR TITLE
Add edge depth parameter

### PR DESCRIPTION
**Pull request by linuss**

### CODE REVIEW

File: modules/mod_ginger_base/controllers/controller_rest_resources.erl [hunk 1]
```diff
@@ -102,7 +103,8 @@ to_json(Req, State = #state{mode = document}) ->
         Id = State#state.rsc_id,
         Context = State#state.context,
         Rsc = m_ginger_rest:rsc(Id, Context),
-        Json = jsx:encode(m_ginger_rest:with_edges(Rsc, Context)),
+        {EdgeDepth, _} = string:to_integer(wrq:get_qs_value("depth", "1", Req)),
+        Json = jsx:encode(m_ginger_rest:with_edges(Rsc, EdgeDepth, Context)),
```
[the remark this answers is not in the document]
Comment (rl-king) [reply modules/mod_ginger_base/controllers/controller_rest_resources.erl]: I agree

[the remark this answers is not in the document]
Comment (linuss) [reply modules/mod_ginger_base/controllers/controller_rest_resources.erl]: It is done

File: modules/mod_ginger_base/controllers/controller_rest_resources.erl [hunk 2]
```diff
@@ -193,13 +194,13 @@ trans({Key, Value}, Acc) ->
 supported_search_args() ->
     ["cat", "hasobject", "hassubject", "sort"].
 
-rsc(Id, Context, IncludeEdges) ->
+rsc(Id, Context, EdgeDepth) ->
```
[the remark this answers is not in the document]
Comment (linuss) [reply modules/mod_ginger_base/controllers/controller_rest_resources.erl]: Done!

**Comment by ddeboer** [comment]
@linuss What are your use cases for this? We decided earlier that we should prefer doing some more HTTP calls over increasing the API’s complexity.

**Comment by rl-king** [comment]
Although I approved this, I agree with David, let's stick to our initial plan

**Pull request closed without merging**